swift/Samples/SampleApp/Sources/CrossFadeContainer.swift

```diff
@@ -0,0 +1,79 @@
+import Workflow
+import WorkflowUI
+
+
+struct CrossFadeScreen: Screen {
+    private var screenType: Any.Type
+    var screen: AnyScreen
+    var key: String
+
+    init<ScreenType: Screen>(_ screen: ScreenType, key: String = "") {
```

Comment:
I really dislike what unnamed parameters do to line-wrapping. Unless this exclusively gets used in a single-line context I’d add that parameter name back.

Reply:
Updated to use `base:` as your example showed. I'm good with it.